exonum-java-binding-qa-service/src/test/java/com/exonum/binding/qaservice/transactions/TestContextBuilder.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2018 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.exonum.binding.qaservice.transactions;
+
+import com.exonum.binding.common.crypto.PublicKey;
+import com.exonum.binding.common.hash.HashCode;
+import com.exonum.binding.storage.database.Fork;
+import com.exonum.binding.transaction.TransactionContext;
+
+public final class TestContextBuilder {
```

Comment:
(applicable for each TestContextBuilder) I am not sure I understand why this class is needed as a builder and not
  as a single method `newContext` that returns a preconfigured `TransactionContext.Builder`
  that already has all required methods (txMessageHash & authorPk).